src/intent/detector.py

```
@@ -0,0 +1,28 @@
+    """
+    Extracts high-level installation intents from natural language requests.
+    """
+
+    COMMON_PACKAGES = {
+        "cuda": ["cuda", "nvidia toolkit"],
+        "pytorch": ["pytorch", "torch"],
+        "tensorflow": ["tensorflow", "tf"],
+        "jupyter": ["jupyter", "jupyterlab", "notebook"],
+        "cudnn": ["cudnn"],
+        "gpu": ["gpu", "graphics card", "rtx", "nvidia"]
+    }
```

Ruff (0.14.4)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)